lib/plugins/aws/deploy/lib/initializeResources.js

```diff
@@ -1,63 +1,27 @@
 'use strict';
 
-const _ = require('lodash');
 const path = require('path');
 const BbPromise = require('bluebird');
 
 module.exports = {
   initializeResources() {
-    const coreCFTemplate = this.serverless.utils.readFileSync(
+    // save the custom resources (e.g. Resources, Outputs etc.) the user has added
+    // so that they can be merged in later on
+    this.serverless.service.customProviderResources = {};
+
+    if (this.serverless.service.resources && this.serverless.service.resources.Resources) {
```

Comment:
ah but that would be a change in functionality which we do in another PR. Alright